packages/devtools_app/lib/src/debugger/flutter/debugger_screen.dart

```diff
@@ -147,28 +162,42 @@ class DebuggerScreenBodyState extends State<DebuggerScreenBody>
       },
     );
 
-    return Split(
-      axis: Axis.horizontal,
-      initialFractions: const [0.25, 0.75],
-      children: [
-        OutlinedBorder(child: debuggerPanes()),
-        Column(
-          children: [
-            DebuggingControls(controller: controller),
-            const SizedBox(height: denseRowSpacing),
-            Expanded(
-              child: Split(
-                axis: Axis.vertical,
-                initialFractions: const [0.74, 0.26],
+    return Shortcuts(
+      shortcuts: <LogicalKeySet, Intent>{
+        LogicalKeySet(LogicalKeyboardKey.control, LogicalKeyboardKey.keyP):
+            FilterLibraryIntent(_libraryFilterFocusNode, controller),
+      },
+      child: Actions(
+        actions: <Type, Action<Intent>>{
+          FilterLibraryIntent: FilterLibraryAction(),
+        },
+        child: Focus(
+          autofocus: true,
```

Comment:
focusing the whole debugger page is a bit arbitrary. Alternatively I could autofocus, say, the "pause" button which is first on the screen? 
<img width="868" alt="Screen Shot 2020-05-05 at 12 46 58 PM" src="https://user-images.githubusercontent.com/3227/81109168-8987b180-8ece-11ea-996d-71a9117bea75.png">

Reply:
That seems reasonable - nothing else on that screen jumps out as a good first focus candidate.

Reply:
This seemed straightforward but I ran into more autofocus issues so I'm going to skip it for now.